lib/sgov/sgov-adapter.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {SgovAdapter} from "./sgov-adapter";
+import {httpFetch} from "../io/fetch/fetch-nodejs";
+import {PimClass} from "../pim/model";
+import {CimAdapter, IriProvider, PrefixIriProvider} from "../cim";
+import {FetchOptions} from "../io/fetch/fetch-api";
+import {CoreResourceReader} from "../core";
+
+let iriProvider: IriProvider;
+let adapter: CimAdapter;
+
+beforeAll(() => {
+  iriProvider = new PrefixIriProvider();
+  adapter = new SgovAdapter("https://slovník.gov.cz/sparql", httpFetch);
+  adapter.setIriProvider(iriProvider);
+});
+
+test("SgovAdapter.search()", async () => {
+  const query = "řidič";
+  const result = await adapter.search(query);
+  expect(result.map(cls => cls.pimHumanLabel?.cs)).toContain("Řidičský průkaz České republiky");
+}, 10 * 60 * 1000);
+
+describe("SgovAdapter.getClass()", () => {
+  test("existing class", async () => {
+    const query = "https://slovník.gov.cz/veřejný-sektor/pojem/fyzická-osoba";
+    const cls = await adapter.getClass(query) as PimClass;
+
+    expect(cls.pimHumanLabel?.en).toEqual("Natural Person");
+    expect(cls.pimHumanDescription?.en).toEqual("Natural Person is a human as a legal subject.");
+  }, 10 * 60 * 1000);
```

Comment:
I think .. is there a way how to set this up-front? Also if an external service is used we may need to mark those tests somehow as it may not be save to run them and they may fail without an obvious reason.

Reply:
Agree. The only thing I can think of right now is Github Actions with a real triplestore. But that won't work on a client.